src/config/inputmethod.swift

```diff
@@ -120,11 +120,13 @@ struct InputMethodConfigView: View {
             }
             .onMove { indices, newOffset in
               group.inputMethods.move(fromOffsets: indices, toOffset: newOffset)
+              viewModel.save()
             }
           }
         }
         .onMove { indices, newOffset in
```

Comment:
On my side group is not movable. Do you experience the same? If not, I think better to remove this `onMove`.

Reply:
Removed. BTW, we should officially not support group reorder. The first group is definitionally the currently active group, so changing group order can have surprising effects.